Gemfile

```diff
@@ -1,12 +1,13 @@
 source "http://rubygems.org"
 
-ruby '2.1.6'
+# TODO: remove me
+# ruby '2.1.6'
```

Comment:
This might cause a problem for us. 

We have a server that listens to merge events from `master` to publish the site to `gh-pages`. I recall much frustration with Ruby versions and getting the setup to work.

I also remembered that nanoc 4 requires 2.2.0 or higher, right? Then I'll definitely have to look into that more closely before the upgrade.